specification/resources/resource-manager/Microsoft.Resources/stable/2024-07-01/resources.json

```diff
@@ -5440,19 +5521,13 @@
       },
       "description": "Deployment on error behavior with additional details."
     },
-    "DeploymentValidateResult": {
+    "DeploymentValidateError": {
```

Comment:
Might 'DeploymentValidationError' sound better?

Reply:
I'll restore it back, I can do `DeploymentValidationResult` since it holds information about "error" or successful response (`DeploymentPropertiesExtended`) as well.